src/components/CategorySelector/index.js

```diff
@@ -1,31 +1,16 @@
 import React from 'react';
-import { Link as RouterLink, useRouteMatch } from 'react-router-dom';
-import { Box, Grid, Link, Typography } from '@material-ui/core';
+import { Grid } from '@material-ui/core';
 import { categories } from '../../helpers/constants';
-import { SquareButton } from '../SquareButton';
+import { CategoryButton } from '../CategoryButton';
 
 export const CategorySelector = ({ availableCategories }) => {
-  let { url } = useRouteMatch();
-
-  const categoriesList =
-    !!availableCategories && Array.isArray(availableCategories)
-      ? Object.values(categories).filter((c) =>
-          availableCategories.includes(c.key)
-        )
-      : Object.values(categories);
+  const categoriesList = availableCategories.sort().map(name => categories[name]);
```

Comment:
If `availableCategories` either not there or is an empty array this will throw an error. We default the `availableCategories` to be an empty array, but since we aren't using PropTypes to validate props (which we probably should start doing), I wonder if it would be best to still check the variable?

I'm fine either way for the above, it just currently breaks the storybook page for demoing this component 😢.